README.md

```diff
@@ -1,2 +1,22 @@
 # couchdb
 Minimalist CouchDB client
+
+## Understanding.
+
+client.go --> represents the couchdb client for the others to use.
```

Comment:
maybe wrap the files in `**FILE_NAME**` as well to get

- **client.go** - represents the couchdb client for the others to use.